devex/cross-nuget/README.md

```diff
@@ -0,0 +1,136 @@
+# Cross-Platform Open Enclave SDK NuGet Package
+
+This directory contains the scripts used to build a cross-platform NuGet package
+for the Open Enclave SDK. The resulting package may be used on Windows to build
```

Comment:
Suggest saying the existence of this package is temporary and issue #3631 tracks unifying this with the other nuget package.

Reply:
Updated.